package.json

```diff
@@ -2,20 +2,28 @@
   "name": "solana-workbench",
   "productName": "SolanaWorkbench",
   "description": "Solana workbench app for making development on Solana better",
+  "version": "0.0.1",
```

Comment:
OK, this one matters - the version, and other metadata should be copied from the deleted release/app/package.json to here - as we've already released 0.0.1, 0.0.2, and 0.0.3. if the other meta, like license can come over too, that would be nice.

Reply:
Oh, I didn't realize there were multiple package.jsons. I should have known that!

Reply:
yeah, this is the kind of stuff that's why i'm thrilled that you've deleted the .erb/ stuff :)